app/templates/events/event_list.html

```diff
@@ -51,77 +51,123 @@ <h4><strong>Term</strong></h4>
-  <div class="accordion" id="categoryAccordion">
-    {% for program,events in categoryPrograms.items() %}
-      <div class="accordion-item">
-        <h2 class="accordion-header" id="accordion__header_{{program}}">
-          <button class="accordion-button collapsed"
-                  type="button"
-                  data-bs-toggle="collapse"
-                  data-bs-target="#accordion__body_{{program}}_num_{{ loop.index }}"
-                  aria-expanded="true"
-                  aria-controls="accordion__body_{{program}}" >
-                  {{program.programName}}
-          </button>
-        </h2>
-        <div id="accordion__body_{{program}}_num_{{ loop.index }}"
-              class="accordion-collapse collapse"
-              aria-labelledby="accordion__header_{{program}}"
-              data-bs-parent="#categoryAccordion">
-          <div class="accordion-body">
-            <table class="table table-striped">
-              <thead>
-                <tr>
-                  <th scope="col">Event Name</th>
-                  <th scope="col">Date</th>
-                  <th scope="col">Time</th>
-                  <th scope="col">Location</th>
-                  <th scope="col">Invitation</th>
-                </tr>
-              </thead>
-              <tbody>
-                  {% for event in events %}
-                    <tr>
-                      <td> <a href="/event/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
-                      <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
-                      <td>{{event.timeStart.strftime('%I:%M %p')}}</td>
-                      <td>{{event.location}}</td>
-                      {% if (user.isCeltsAdmin) or (user.isCeltsStudentStaff) %}
-                        <td><i class="bi bi-envelope-fill"> Email</i></td>
-                        {# FIXME: the Email icon should be clickable and lead to email template #}
-                      {% else %}
-                        {% if (event.startDate < currentTime.date()) or (event.timeStart < currentTime.time() and event.startDate == currentTime.date())%}
-                          <td><button type="button" class="btn btn-primary" disabled>RSVP</button></td>
-                        {% elif event.id in rsvpedEventsID %}
-                          <td><button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{program.id}}, {{event.id}})">Remove RSVP</button></td>
+{% macro createTable(events) %}
+{% if events.exists() %}
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th scope="col">Event Name</th>
+        <th scope="col">Date</th>
+        <th scope="col">Time</th>
+        <th scope="col">Location</th>
+        <th scope="col">Invitation</th>
+      </tr>
+    </thead>
+    <tbody>
+
+        {% for event in events %}
+          <tr>
+            <td> <a href="/{{event.program_id}}/{{event.id}}/edit_event" class="link-primary">{{event.name}}</a></td>
```

Comment:
This url is wrong. Make sure when you're replacing a big block, that you identify changes that were made in that block